test/test_nn.py

```diff
@@ -7699,6 +7701,39 @@ def test_adaptive_log_softmax(self):
         out = asfm.predict(x)
         self.assertEqual(out, asfm.log_prob(x).argmax(dim=1))
 
+    def test_transformer_number_match(self):
+        # Train the simple copy task.
+        V = 11
+        num_encoder_layers = 1
+        num_decoder_layers = 1
+        d_model = 16  # embedding
+        nhead = 2  # number of heads
+        dropout = 0.20
+        d_ff = 64
+        batch_size = 400
+        milliseconds = int(round(time.time() * 1000))
+        torch.manual_seed(milliseconds)
+
+        model = Transformer(V, V, d_model=d_model, nhead=nhead, num_encoder_layers=num_encoder_layers, 
+                            num_decoder_layers=num_decoder_layers, d_ff=d_ff)
```

Comment:
What is "d_ff"? The name is a bit nondescript.

Reply:
Will make some changes in the next commit.